content/docs/guides/mediawiki-tabular-data.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Comparison with MediaWiki Tabular Data
+sidebar:
+  order: 3
+---
+
+<table>
+  <tr>
+    <th>Authors</th>
+    <td>Jakob Voß</td>
+  </tr>
+</table>
+
+[MediaWiki](https://www.mediawiki.org/) is the software used to run Wikipedia and related projects of the Wikimedia Foundation, including the media file repository [Wikimedia Commons](https://commons.wikimedia.org/). Commons hosts mostly images but also some records with tabular data. The [MediaWiki Tabular Data Model](https://www.mediawiki.org/wiki/Help:Tabular_data) was inspired by Data Package version 1 but it slightly differs from current Data Package specification, as described below.
+
+## Comparison
+
+A [MediaWiki tabular data page](https://www.mediawiki.org/wiki/Help:Tabular_data) describes and contains an individual table of data similar to a [Data Resource](/standard/data-resource/) with inline tabular data. Both are serialized as JSON objects, but the former comes as a page with unique name in a MediaWiki instance (such as Wikimedia Commons).
+
+### Top-level fields
+
+MediaWiki Tabular data has three required and two optional top-level fields. Most of these fields map to corresponding fields of a Data Resource:
+
+Field | Data Resource | Tabular data
```

Comment:
https://datapackage.org/guides/csvw-data-package/#columns compares properties for users that know CSVW, but want to know if it is supported in Data Package. I wonder if the same approach would make sense here? It would allow to combine the table with the differences.

For example (see source for markdown):

MediaWiki Tabular Data property | Data Package support | Details
--- | --- | ---
name (implied from page)* | Yes | As [resource.name](/standard/data-resource/#name)
description | Yes | As [resource.description](/standard/data-resource/#description), but without support for [localized strings](#data-types)
data* | Yes | MediaWiki always requires an array of arrays, while Data Package supports other formats as well as data files referred to by path
license | Yes | As [resource.licenses](/standard/data-resource/#licenses) with a `name` (e.g. `CC0-1.0`)
schema* | Yes | As [resource.schema](/standard/data-resource/#schema), see [further](#schema-properties)
sources | Yes | As [resource.sources](/standard/data-resource/#sources), but without support for Wiki markup

Reply:
If possible, I would also link the MediaWiki Tabular Data property